src/components/theme/View/View.jsx

```diff
@@ -203,8 +203,17 @@ class View extends Component {
    */
   render() {
     const { views } = config;
-    if (this.props.error && this.props.error.code === 301) {
-      const redirect = flattenToAppURL(this.props.error.url).split('?')[0];
+    const status = __SERVER__
+      ? this.props.error?.status
+      : this.props.error?.code;
+    // Checking to see if it's a 3XX HTTP status code. error.status only works on the server
+    if (status?.toString()?.[0] === '3') {
```

Comment:
Not all the `3XX` are redirects (i.e. 304 Not modified), IMHO is better to use something like  `[301, 302, 307, 308].includes(status)`